test/module-test.ts

```diff
@@ -36,7 +36,7 @@ describe('napajs/module', function () {
         it('napa module', () => {
             return napaZone.execute((thisFilePath: string) => {
                 var assert = require("assert");
-                var napaModule = require(thisFilePath + '/module/addon/build/simple-napa-addon.napa');
+                var napaModule = require(thisFilePath + '/../bin/simple-napa-addon.napa');
```

Comment:
Why do we call it a 'sample'?

---
In reply to: [133246133](https://github.com/Microsoft/napajs/pull/41#discussion_r133246133) [](ancestors = 133246133)